### PR TITLE
Documentation: better docstrings

### LINKED ISSUE #72: Documentation: better doc strings
### What is the expected enhancement?
Increase readability of docstrings for tests, classes and arguments.

### PR DESCRIPTION
**Pull request by mickahell**
### Summary
Add documentation to unittests methods

#### Tests files
- [x] commands
- [x] daos
- [x] models
- [x] runners
- [x] utils
- [x] common
- [x] tests_manager

#### Src files
- [x] commands
- [x] daos
- [x] models
- [x] runners
- [x] utils
- [x] manager

Closes #72 
Epic #9

### CODE REVIEW

**Comment by mickahell** [comment]
Hi @IceKhan13 what do you think of that :) ?

File: tests/daos/test_dao.py [hunk 1]
```diff
@@ -58,7 +68,15 @@ def test_repository_insert_and_delete(self):
...
     def test_move_from_tier_to_tier(self):
-        """Tests moving repo from tier to tier."""
+        """Tests moving repo from tier to tier.
+        Function: JsonDao
+                -> move_repo_to_other_tier
+        Args:
+            repo_url: URL for repository
+            source_tier: source tier of repository
+            destination_tier: destination tier
+        Returns: Changed repository
```
Comment (IceKhan13) [inline tests/daos/test_dao.py:78]: no rerturn

File: tests/models/test_configuration.py [hunk 2]
```diff
@@ -36,7 +40,11 @@ def test_save_and_load(self):
         )
 
     def test_python_configuration(self):
-        """Tests python configurations."""
+        """Tests python configurations.
+        Function: RepositoryConfiguration
```
Comment (IceKhan13) [inline tests/models/test_configuration.py:44]: same

File: tests/models/test_execution_summary.py [hunk 3]
```diff
@@ -9,7 +9,10 @@ class TestExecutionSummary(TestCase):
     """Tests for execution summary class."""
 
     def test_qiskit_deprecation_warning_detection(self):
-        """Tests detection of qiskit deprecation logs."""
+        """Tests detection of qiskit deprecation logs.
+        Function: CommandExecutionSummary
```
Comment (IceKhan13) [inline tests/models/test_execution_summary.py:13]: same

**Comment by IceKhan13** [comment]
Google docstring format: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

**Review by IceKhan13 — APPROVED** [review]
LGTM

**Pull request merged**